### ISSUE TITLE
Frescobaldi editor auto-completion needs update

### ISSUE THREAD

**Issue by papioara** [frescobaldi/frescobaldi#1740]
Using Frescobaldi a lot, I have noticed that two frequently used commands used by me are either not present or even outdated:

1. `\tupletSpan:` auto-complete only knows `\tuplet`, and you have to fill in the rest manually
2. `\compressEmptyMeasures`: auto-complete always suggests `\compressFullBarRests`, which throws an error on compilation.

I have checked the contents of $HOME/frescobaldi/frescobaldi_app/autocomplete/completiondata.py, but these commands are not listed there. So my question is: where does the auto-completion load these commands from? From somewhere in LilyPond? If I knew, I could maybe fix it at least on my machine by adding/changing the entries.

I use LilyPond 2.24.2

**Comment by ksnortum** [comment]
The autocomplete function is in python-ly (
https://github.com/frescobaldi/python-ly) in ly/words.py.  The
latest version has those two autocomplete words in it but I don't know if
it has been released yet.

**Comment by papioara** [comment]
Thank you for the hint. I have taken a look at python-ly 0.9.7, but in ly/words.py they do not exist yet. That's what you can download from the python-ly website. Seems to me that you are talking of a newer version which you have seen already, but which is not out yet.

**Comment by papioara** [comment]
In 0..9.8 on Github those words do exist - that is correct. For the time being I did a little tinkering on my machine: I just opened the words.py file in my /usr/lib/python3/dist-packages/ly directory and added/commented out what I wanted. Now my Frescobaldi knows those commands and is amnesiac about /compressFullBarRests. Wonderful! 
Thank you again - you inspired me to find the file Frescobaldi is drawing its 'knowledge' from.
